tests/e2e/e2e-tests/utilscript.sh

```diff
@@ -22,7 +22,7 @@ if [ "$2" = "pvcreate" ]
 elif [ "$2" = "voldelete" ]
 then
     export VOL_ID=$1
-    bx sl  block volume-cancel -f $VOL_ID
+    bx sl  block volume-cancel -f --immediate $VOL_ID
```

Comment:
As discussed you need to do un-authorize before cancel of volume.
bx sl block access-revoke -p <NODE IP> <VOL ID>

Reply:
I ahve added the change but somehow missed pushed the change now